features/dd-sdk-android-rum/src/test/kotlin/com/datadog/android/telemetry/internal/TelemetryEventHandlerTest.kt

```diff
@@ -353,19 +354,28 @@ internal class TelemetryEventHandlerTest {
         }
     }
 
-    @Test
-    fun `𝕄 create config event 𝕎 handleEvent(SendTelemetry) { configuration with tracing settings }`(
-        @Forgery fakeConfiguration: TelemetryCoreConfiguration,
-        @BoolForgery useTracing: Boolean,
-        forge: Forge
+    @ParameterizedTest
+    @MethodSource("tracingConfigurationParameters")
+    fun `𝕄 create config event 𝕎 handleEvent(SendTelemetry) { tracing configuration with tracing settings }`(
+        configuration: Triple<Boolean, TelemetryEventHandler.TracerApi?, String?>,
+        @Forgery fakeConfiguration: TelemetryCoreConfiguration
     ) {
```

Comment:
instead of using `Triple` without clear names it is possible to be more explicit:

```suggestion
    @ParameterizedTest
    @MethodSource("tracingConfigurationParameters")
    fun `𝕄 create config event 𝕎 handleEvent(SendTelemetry) { tracing configuration with tracing settings }`(
        useTracer: Boolean,
        tracerApi: TelemetryEventHandler.TracerApi?,
        tracerApiVersion: String?,
        @Forgery fakeConfiguration: TelemetryCoreConfiguration
    ) {
```

and then in the method provider instead of `Triple` you can use `Arguments.of`